### ISSUE TITLE
Rethink dataType and dataSubtype names

### ISSUE THREAD

**Issue by karawoo** [Sage-Bionetworks/synapseAnnotations#421]
Every time we mention dataSubtype everyone seems to agree that it needs a different name. Opening this issue to rethink the names of the dataType and dataSubtype keys.

**Comment by allaway** [comment]
cc @sgosline for the obligatory `:thinking-face:`

**Comment by allaway** [comment]
Whoops. Did not mean to close.

**Comment by karawoo** [comment]
Darn @allaway I was hoping you'd solved it! 😉

**Comment by sgosline** [comment]
I think this is sort of captured in the 'Level I-4' annotations [in TCGA](https://cancergenome.nih.gov/abouttcga/aboutdata/datalevelstypes) where level 1 is raw and level 4 is processed...

So I agree that subtype isn't the right word - `dataState` is the only value that comes to mind so far...

**Comment by karawoo** [comment]
I think something that conveys different levels is good. `dataLevel`? `dataTier`? Landsat data has different tiers for different levels of processing.

**Comment by allaway** [comment]
I agree that the TCGA levels are probably a good starting point, and
they've explicitly defined many of our commonly used `dataType`s already
(although they track more closely with our `assay`s, I think. It might be a
challenge to define these sets for every dataType.

**Comment by sgosline** [comment]
I think that's ok for now. We can always add more granularity later. I'd stay away from 'Level' since that, to me, is so closely tied to TCGA, but 'dataTier' might work.

**Comment by karawoo** [comment]
`dataProcessingLevel` was also discussed in the meeting today, though the (admittedly lukewarm) favorite seemed to be `dataLevel`

**Comment by kdaily** [comment]
Check out GDC data dictionary here:

https://docs.gdc.cancer.gov/Data_Dictionary/viewer/
